pg_search/tests/index_config.rs

```diff
@@ -641,3 +641,90 @@ fn partitioned_index(mut conn: PgConnection) {
         Err(err) => assert_eq!(err.to_string(), "error returned from database: Creating BM25 indexes over partitioned tables is a ParadeDB enterprise feature. Contact support@paradedb.com for access."),
     };
 }
+
+#[rstest]
+fn drop_schema_cascades_index(mut conn: PgConnection) {
+    // Test that dropping a schema cascades to drop the index as well
+    "CALL paradedb.create_bm25_test_table(table_name => 'index_config', schema_name => 'paradedb')"
+        .execute(&mut conn);
+
+    "CALL paradedb.create_bm25(
+        index_name => 'index_config',
+        table_name => 'index_config',
+        schema_name => 'paradedb',
+        key_field => 'id',
+        text_fields => paradedb.field('description')
+    )"
+    .execute(&mut conn);
+
+    // Drop the schema and ensure the index is also dropped
+    "DROP SCHEMA paradedb CASCADE".execute(&mut conn);
```

Comment:
hmm.  `paradedb` is the `pg_search` extension's schema too.  Maybe that doesn't matter for a test, but it caught my attention as a little weird.

I don't know if we'd instead want to create a schema in which we create a table and an index, just to better model what a user would do.  I sure hope users aren't putting their objects in the `paradedb` schema!